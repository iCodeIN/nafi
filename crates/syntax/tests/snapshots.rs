use nafi_syntax::SourceFile;

#[test]
fn parse_snapshot_tests() {
    insta::glob!("snapshots/*.nafi", |path| {
        let input = std::fs::read_to_string(path).unwrap();
        insta::assert_debug_snapshot!(SourceFile::parse(&input).syntax_tree());
    });
}
