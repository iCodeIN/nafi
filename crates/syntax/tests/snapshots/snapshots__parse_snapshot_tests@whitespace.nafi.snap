---
source: crates/syntax/tests/snapshots.rs
expression: "SourceFile::parse(&input).syntax_tree()"
---
SourceFile@0..2
  Whitespace@0..2 " \n"

